test/MainEvent.cxx

```diff
@@ -131,9 +131,12 @@ int main(int argc, char **argv)
    Int_t branchStyle = 1; //new style by default
    if (split < 0) {branchStyle = 0; split = -1-split;}
 
+   ROOT::EnableImplicitMT(4);
```

Comment:
Would you prefer that it is automatically enabled or add it as a command line flag?